internal/service/streaminstance/data_source_stream_instances_test.go

```diff
@@ -44,40 +41,40 @@ func TestAccStreamDSStreamInstances_withPageConfig(t *testing.T) {
 		CheckDestroy:             acc.CheckDestroyStreamInstance,
 		Steps: []resource.TestStep{
 			{
-				Config: streamInstancesWithPageAttrDataSourceConfig(orgID, projectName, instanceName, region, cloudProvider),
-				Check:  streamInstancesAttributeChecks(dataSourceName, admin.PtrInt(2), admin.PtrInt(1), 0),
+				Config: streamInstancesWithPageAttrDataSourceConfig(projectID, instanceName, region, cloudProvider, 1000),
```

Comment:
could we give that 1000 a variable name?

Reply:
yes, extracted